### PR TITLE
record ExecPlan total RowsRead, BytesScan

### PR DESCRIPTION
**Pull request by xzxiong**
## What type of PR is this?

- [ ] API-change
- [ ] BUG
- [x] Improvement
- [ ] Documentation
- [ ] Feature
- [ ] Test and CI
- [ ] Code Refactoring

## Which issue(s) this PR fixes:

issue https://github.com/matrixorigin/matrixone/issues/5503

## What this PR does / why we need it:
statistics total RowsRead and BytesScan from ExecPlan, record it in table `system.statement_info` column `rows_read` and `bytes_scan`
- sum `Input Rows` and `Input Size` statistics from node `External Scan` and `Table Scan`

### CODE REVIEW

**Review by daviszhen — APPROVED** [review]

**Review by qingxinhome — APPROVED** [review]
It's very beautiful

File: pkg/sql/plan/explain/explain_node.go [hunk 1]
```diff
@@ -33,6 +33,9 @@ func NewNodeDescriptionImpl(node *plan.Node) *NodeDescribeImpl {
 	}
 }
 
+var TableScan = "Table Scan"
```
Comment (ouyuanning) [inline pkg/sql/plan/explain/explain_node.go:36]: why not const?

File: pkg/sql/plan/explain/marshal_query.go [hunk 2]
```diff
@@ -527,6 +527,9 @@ func (m MarshalNodeImpl) GetNodeLabels(options *ExplainOptions) ([]Label, error)
 	return labels, nil
 }
 
+var InputRows = "Input Rows"
```
Comment (ouyuanning) [inline pkg/sql/plan/explain/marshal_query.go:530]: const?

**Review by fengttt — COMMENTED** [review]
I believe we said we only log plan in AP queries.   Cannot afford this for short TP queries.
Is this true?

Just want to check and verify.

**Comment by xzxiong** [comment]
> I believe we said we only log plan in AP queries. Cannot afford this for short TP queries. Is this true?
> 
> Just want to check and verify.

TP also record those statistics data.

1. Every query record original data in ExecPlan, `trace.Statement` hold its ExecPlan obj in mem.
2. When we need to  serialize ExecPlan as json, we will check condition `statement.duraiton >= config.LongQueryTIme`
   - true: get real json str
   - false: get dummy json like `{"message":"sql query no record execution plan","success":false, ...}`
3. right now, we only serialize ExecPlan while we need to export statement info into s3.

So, we can record all query's total RowsRead, BytesScan.

**Review by ouyuanning — APPROVED** [review]

**Review by aunjgr — APPROVED** [review]

**Comment by xzxiong** [comment]
@mergify refreash

**Comment by xzxiong** [comment]
@mergify requeue

**Review by fengttt — APPROVED** [review]

**Comment by fengttt** [comment]
@mergifyio refresh

**Comment by xzxiong** [comment]
@Mergifyio refresh

**Pull request merged**